### PR TITLE
Update va-progress-bar aria-label when percent is used to round

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/34279

## Testing done
- [ ] JAWS
- [x] NVDA
- [x] VoiceOver

## Acceptance criteria
- [x] Format aria-label percent value to 0 significant figures

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by pdavies88 — APPROVED** [review]

File: packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class VaProgressBar {
   }
 
   render() {
-    const { label = `${this.percent}% complete`, percent } = this;
+    const { label = `${this.percent.toFixed(0)}% complete`, percent } = this;
```
Comment (bkjohnson) [inline packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx:49]: Can we get a test showing that the `aria-label` gets truncated to an integer?

Also, we're using `percent` for the `aria-valuenow` attribute as well. Can @CourtneyChambers77 check to see if that is the same kind of a11y annoyance as the `aria-label` having too many decimals?

https://github.com/department-of-veterans-affairs/component-library/blob/333cf42aaddc429945fd1bd5cb439532e697246e/packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx#L57

**Comment by andresriveratoro** [comment]
Latest commit: https://60f9b557105290003b387cd5-eqnctkhxlt.chromatic.com/

**Comment by andresriveratoro** [comment]
@CourtneyChambers77 https://60f9b557105290003b387cd5-eqnctkhxlt.chromatic.com/iframe.html?id=components-va-progress-bar--default&viewMode=story

File: packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx (hunk 1 above)
Comment (andresriveratoro) [reply packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx:49]: NVDA is reading out the entire percent including decimals for `aria-valuenow`.

File: packages/storybook/stories/va-progress-bar.stories.jsx [hunk 2]
```diff
@@ -17,8 +17,8 @@ const Template = ({ 'enable-analytics': enableAnalytics, label, percent }) => (
 
 const defaultArgs = {
   'enable-analytics': false,
-  'label': 'Add a label here',
-  'percent': 35,
+  'label': undefined,
+  'percent': 48.123456,
```
Comment (andresriveratoro) [inline packages/storybook/stories/va-progress-bar.stories.jsx:21]: I made these changes to make it easier for Courtney to test accessibility. I can revert them before merging this PR if requested.

**Review by pdavies88 — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]
:+1:

**Pull request merged**